build/buildspec.docker.dev-ent.yml

```diff
@@ -23,20 +22,6 @@ phases:
       - chmod 600 ~/.ssh/ssh_key
       - eval "$(ssh-agent -s)"
       - ssh-add ~/.ssh/ssh_key
-      - (cd && go get github.com/onsi/ginkgo/ginkgo)
-      - (cd && go get github.com/golang/mock/mockgen)
-      - docker pull confluentinc/cp-zookeeper:latest
-      - docker pull confluentinc/cp-kafka:7.0.0
-      - docker pull minio/minio:latest
-      - docker pull redis:alpine3.14
-      - docker pull postgres:11-alpine
```

Comment:
We would need to communicate with @rudderlabs/infra  too about these changes we are making with regards to buildspec files. Last time they got reverted as we directly changed on AWS code build and not through the terraform script

Reply:
Do we include buildspec files on terraform or aws build configurations?

I haven't changed configurations for aws console this time.